region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -1918,6 +1983,45 @@
         "Knocking the crab off while it is on the the door or ceiling will not work."
       ]
     },
+    {
+      "link": [4, 4],
+      "name": "Sciser Farm (1 Sciser, Fast Cycle)",
+      "requires": [
+        {"resetRoom": {"nodes": [3]}},
+        {"or": [
+          "canCrouchJump",
+          "h_underwaterCrouchJumpWithFlashSuit"
+        ]},
+        {"or": [
+          {"and": [
+            "Plasma",
+            {"cycleFrames": 150}
+          ]},
+          {"and": [
+            "Wave",
```

Comment:
This one is expert, but since you are practicing it over and over it may be very hard, but higher than plasma

Reply:
~~I don't think it works with gravity~~.  Maybe you'd get the 3 crab kills in that case anyway.
It's just unintuitive.  You can diagonal shot kill it with or without gravity.